### PR TITLE
Add copyright entry

### LINKED ISSUE #9: Add Copyright info
I tried to submit to apple podcasts, though there was an issue that we need to define the copyright on the feed.

image

### PR DESCRIPTION
**Pull request by benfoxall**
resolves #9

### CODE REVIEW

File: feed.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@ layout: compress
     <itunes:explicit>{{ site.feed.explicit }}</itunes:explicit>
     <itunes:image href="{{ urlbase }}{{ site.feed.image }}" />
     <itunes:category text="{{ site.feed.category }}"/>
+    <copyright>Copyright 2020-2021 Remote Hack Space</copyright>
```
Comment (lpmi-13) [inline feed.xml]: could this be defined in jekyll config instead of a string?

Comment (benfoxall) [reply feed.xml]: Oh, absolutely! That's a **way** better place to define it.

Comment (omgmog) [reply feed.xml]: Could probably make it generate this dynamically using the first and last episode dates to get the years, and pulling the "Remote Hack" bit straight out of the config

Comment (lpmi-13) [reply feed.xml]: Max off the top rope with an outstanding suggestion!

**Comment by omgmog** [comment]
💁 28fda16

**Comment by benfoxall** [comment]
### Amazing!!  **SO DYNAMIC!** 🥰

**Pull request merged**